code/modules/clothing/head/jobs.dm

```diff
@@ -51,6 +51,14 @@
 
 	dog_fashion = null
 
+/obj/item/clothing/head/caphat/beret/white
+	name = "captain's white beret"
+	desc = "A white beret fit for a leader."
+	icon_state = "beret_captain_white"
+	dynamic_hair_suffix = ""
+
+	dog_fashion = null

```

Comment:
dynamic_hair_suffix and dog_fashion don't need to be defined as they are inherited from /obj/item/clothing/head/caphat/beret